packages/paste-website/src/component-examples/CodeBlockExamples.ts

```diff
@@ -0,0 +1,41 @@
+const multiLineCodeSnippet = `
+<?php
+require __DIR__ . '/vendor/autoload.php';
+use Twilio\Rest\Client;
+
+// Your Account SID and Auth Token from twilio.com/console
+$account_sid = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+$auth_token = 'your_auth_token';
+// In production, these should be environment variables. E.g.:
+// $auth_token = $_ENV["TWILIO_AUTH_TOKEN"]
+
+// A Twilio number you own with SMS capabilities
+$twilio_number = "+15017122661";
+
+$client = new Client($account_sid, $auth_token);
+$client->messages->create(
+    // Where to send a text message (your cell phone?)
+    '+15558675310',
+    array(
+        'from' => $twilio_number,
+        'body' => 'I sent this message in under 10 minutes!'
+    )
+);`;
+
+export const multiLine = `
```

Comment:
Current issue: Either Gatsby or the live preview is having a really hard time with the fact that I'm trying to pass strings that look like code as the child of CodeBlock. I finally got the single line example to work, but the multi-line one is still having issues with the `<?php` part.

I have tried:
* creating the example inline in the markdown, similar to how the single line example is
* putting the contents of the multiLIneCodeSnippet var directly into the multiLine string
* and the current solution

Not really sure how to get around this... I could always change the example, but this is the one from [the Match website](https://match.twilio.design/components/snippet/) so I thought I should keep it.